code/index.html

```diff
@@ -1,21 +1,55 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="favicon.ico" type="image/x-icon"/>
+    <title>Kreis4 Flower Shop - Order Flowers Online</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
 
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
-    <!-- If you're using google fonts, that should be added here -->
-  </head>
-  <body>
-    <!-- 
-      Write your code here inside the body to make it appear in the browser. 
-      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
+    <menu>
+        <a href="./index.html">Home</a>
+        <a href="./our-flowers.html">Flowers</a>
+        <a href="./subscription.html">Subscription</a>
+        <a href="./faqs.html">FAQs</a>
+        <a href="./about.html">About Us</a>
+    </menu>
 
-      <img src="example-recipe-background.png" alt="Banana Bread">
+    <header>
+    <h1>Welcome to Kreis4 Flowers<br>Fresh Flowers All Year Round</h1>
+    </header>
```

Comment:
We should try to avoid the `br` element to write more semantic HTML. You could create two different `h1`-tags and remove the margins of it with CSS :)

Reply:
Also, remember to indent your code when an element contains another element:
```
<header>
    <h1>Welcome to Kreis4 Flowers</h1>
    <h1>Fresh Flowers All Year Round</h1>
</header>
```